data/state1/ACCESSCONTROL-ROLEACTIONS/roleactions.json

```diff
@@ -85,6 +85,900 @@
       "actionid": 13,
       "actioncode": "",
       "tenantId": "state1"
+    },
+        {
+      "rolecode": "SUPERUSER",
+      "actionid": 6,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "ESTIMATE_CREATOR",
+      "actionid": 14,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "ESTIMATE_VIEWER",
+      "actionid": 10,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "ESTIMATE_VIEWER",
+      "actionid": 16,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "ESTIMATE_VIEWER",
+      "actionid": 58,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "ESTIMATE_VIEWER",
+      "actionid": 60,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "ESTIMATE_VIEWER",
+      "actionid": 61,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "ESTIMATE_VIEWER",
+      "actionid": 66,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "ESTIMATE_VIEWER",
+      "actionid": 67,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "ESTIMATE_VIEWER",
+      "actionid": 68,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "ESTIMATE_VIEWER",
+      "actionid": 77,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "ESTIMATE_VIEWER",
+      "actionid": 89,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "ESTIMATE_VERIFIER",
+      "actionid": 17,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "TECHNICAL_SANCTIONER",
+      "actionid": 17,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "ESTIMATE_APPROVER",
+      "actionid": 17,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
+      "rolecode": "ESTIMATE_CREATOR",
+      "actionid": 17,
+      "actioncode": "",
+      "tenantId": "state1"
+    },
+    {
```

Comment:
"rolecode": "ESTIMATE_CREATOR",
      "actionid": 18,
      "actioncode": "",
      not required